addons/link_tracker/models/link_tracker.py

```diff
@@ -195,6 +197,8 @@ def create(self, vals_list):
     def search_or_create(self, vals):
         if 'url' not in vals:
             raise ValueError(_('Creating a Link Tracker without URL is not possible'))
+        if vals['url'].startswith(('?', '#')):
```

Comment:
Arguable could cover `/*?` and `/*#` but I'm sure there's plenty of ways to mess this one up so the current one could/should be enough I guess

Reply:
an URL `/?` would lead to the root page hence note loop